zone/zonedb.cpp

```diff
@@ -1980,7 +1980,7 @@ const NPCType *ZoneDatabase::LoadNPCTypesData(uint32 npc_type_id, bool bulk_load
 
 	DataBucket::BulkLoadEntities(DataBucketLoadType::NPC, npc_ids);
 
-	if (bulk_load) {
+	if (!npc_faction_ids.empty()) {
```

Comment:
Where do we use `bulk_load`?

Reply:
I added that for awhile to isolate what was being loaded so I could test.  Not needed here.  I added the empty check just to reduce instrumentation and an unneeded call.